flow/raster_cache.cc

```diff
@@ -259,4 +259,35 @@ void RasterCache::SetCheckboardCacheImages(bool checkerboard) {
   Clear();
 }
 
+void RasterCache::TraceStatsToTimeline() const {
+#if FLUTTER_RUNTIME_MODE != FLUTTER_RUNTIME_MODE_RELEASE
+
+  size_t layer_cache_count = 0;
+  size_t layer_cache_bytes = 0;
+  size_t picture_cache_count = 0;
+  size_t picture_cache_bytes = 0;
+
+  for (const auto& item : layer_cache_) {
+    const auto dimensions = item.second.image.image_dimensions();
+    layer_cache_count++;
+    layer_cache_bytes += dimensions.width() * dimensions.height() * 4;
+  }
+
+  for (const auto& item : picture_cache_) {
+    const auto dimensions = item.second.image.image_dimensions();
+    picture_cache_count++;
+    picture_cache_bytes += dimensions.width() * dimensions.height() * 4;
+  }
+
+  FML_TRACE_COUNTER("flutter", "RasterCache",
```

Comment:
This does not seem to be limited to COUNTER. Maybe rename it to something like `FML_TRACE_KEY_VALUE_PAIRS` to indicate that we can plugin any VALUE that can be converted by `std::to_string`?

Reply:
This will map to `TRACE_COUNTER` on Fuchsia which is only limited to counts on Fuchsia. I'd rather use the qualified subset of functionality.